### PR TITLE
Support the latest Discord major update

### PR DESCRIPTION
**Pull request by d0gkiller87**

### CODE REVIEW

**Comment by Florian-Mahlberg** [comment]
Looks good, only "problem" is, there pops up a small windows in the top left (disappears after a couple of seconds) with the info, that you dont have spotify premium. Idk if that needs to be fixed?
Picture of the bug

**Comment by d0gkiller87** [comment]
> Looks good, only "problem" is, there pops up a small windows in the top left (disappears after a couple of seconds) with the info, that you dont have spotify premium. Idk if that needs to be fixed? Picture of the bug

Ah I didn't notice it at all at the first place. We can confirm it's reproducible and I will try to look into it.

**Comment by Florian-Mahlberg** [comment]
It's not that important since it's not that big... Maybe not that important

**Comment by d0gkiller87** [comment]
Hi @Florian-Mahlberg, after some test I found out:
1) The misplaced message in full is actually `You're already along for this ride.` (`USER_ACTIVITY_ALREADY_SYNCING`) instead of saying you don't have premium.
2) It happens to my premium account as well (without the plugin installed) thus, it might be some weird CSS bugs of the client.
3) It's not always reproducible. Some of my friend always see it while some always don't (it depends on the `already syncing` status, which I completely have no idea how to trigger lol).

TL;DR; I can't help fix it.

**Comment by Florian-Mahlberg** [comment]
Hey,
Yeah I couldn't reproduce it all of the time as well... lets just leave it how it is and maybe Discord fixes it itself?
Btw I think the `already syncing` should come from the button to listen along, so you can't press twice on it

**Comment by d0gkiller87** [comment]
Yeah, hopefully some might fix it in the future.

**Comment by NekoSuneVR** [comment]
far as i know plugin working where it is right now, im able join people no premium required.

that code posted did fix it but i havent tried premium since i never pay spotify even i used be on family package over my friend cross USA and him completely stopped Family package and i was like nope not paying it anymore after what him did

and cant relink it back USA coz now Region locked in UK now.

so i normally use this plugin with SpotX as Bypass ads and stuff.

**Comment by Florian-Mahlberg** [comment]
Then you would be kinda the chosen one, since you normally need spotify premium

**Comment by ordinall** [comment]
Nice work @d0gkiller87, merged to patch branch. I am just gonna do a few changes and then merge to master.
